### ISSUE TITLE
WarioWare Twisted

### ISSUE THREAD

**Issue by elgarlic** [spiritualized1997/openFPGA-GBA#2]
Since this is mentioned in the description, I thought I would try out the US rom while the Japanese (affordable) cartridge is inserted. No gyro function is working. Should it be? Thanks!

**Comment by RewoundVHS** [comment]
It is explicitly stated that WarioWare Twisted will not work because the Analogue Pocket lacks a gyroscope.

**Comment by elgarlic** [comment]
What it explicitly says is "Warioware Twisted would be if there was a usable accelerometer...". Obviously, it does not specify what is considered usable. I provided an accelerometer, and was wondering if that should be sufficient.

**Comment by hiddenindians** [comment]
Would the DS Motion Sensor (The one shipped with Tony Hawk Motion) work as a proxy gyro?

**Comment by HeyItsLollie** [comment]
Could gamepads like the DualShock 4 or DualSense be viable options for gyro/accelerometers, while playing via the Dock?

**Comment by Mi5KL** [comment]
Yeah, I've been wondering this too. If one game's gyro stuff can extend to the whole set then it would mean you could pick up a super cheap copy of Korokoro Puzzle Happy Panechu and be able to play the english versions of Yoshi and WarioWare, though I'm not really sure if they all work exactly the same.
